client/modules/IDE/hooks/useWhatPage.js

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable no-useless-escape */
```

Comment:
It's considering the `\/` as a "useless" escape because you are using `new RegExp(string)` rather than creating the regex like `const myRe = /text/;`.  The `/` doesn't have any special meaning when inside a `string` so you don't need to escape it.